chef/data_bags/crowbar/template-network.json

```diff
@@ -8,7 +8,9 @@
       "enable_tx_offloading": true,
       "mode": "single",
       "teaming": {
-        "mode": 1
+        "mode": 1,
+        "miimon": 100,
+        "xmit_hash_policy": "layer2"
```

Comment:
@vuntz: Was that referring to ```xmit_hash_policy``` only? Or would you also like to see ```miimon``` changable per-network? AFAICS the current code only allows ```xmit_hash_policy```.

```miimon``` is only looked up in the "global" setting if I read the code correctly.